pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -155,8 +157,8 @@ then an algebra student simplifies @tt{x + 1} as follows:
 
 @verbatim{  x + 1 = 10 + 1 = 11}
 
-Racket works much the same way, in that a set of @tech{top-level
-variables} are available for substitutions on demand during
+Racket works much the same way, in that a set of @deftech{top-level
+variables} (see also @secref["vars-and-locs"]) are available for substitutions on demand during
```

Comment:
This doesn’t seem like the right place to define the term “top-level variable” to me, and indeed, as-written, it doesn’t define top-level variables at all. I think forward references in the evaluation model section are fine, since the evaluation model is supposed to serve as a dense, comprehensive reference by design, not a guide. Forward references seem like a bad idea in, say, The Racket Guide, but I’d keep the definition in the appropriate section here and maintain the forward reference.

Reply:
It's odd, then, that the section is titled ``Top-Level Variables.'' But I'll make this change.